modules/server/src/test/scala/scaladex/server/route/BadgesTests.scala

```diff
@@ -22,17 +25,6 @@ class BadgesTests extends ControllerBaseSuite with BeforeAndAfterAll {
   def insertCats(): Future[Unit] =
     Cats.allArtifacts.map(database.insertArtifact(_, Seq.empty, now)).sequence.map(_ => ())
 
-  it("should provide a concise summary of latest versions") {
-    Badges.summaryOfLatestVersions(
-      Map(
-        `2.11` -> Seq(`7.0.0`, `7.1.0`),
-        `2.12` -> Seq(`7.0.0`, `7.1.0`, `7.2.0`),
-        `2.13` -> Seq(`7.0.0`, `7.1.0`, `7.2.0`, `7.3.0`),
-        `3` -> Seq(`7.2.0`, `7.3.0`)
-      )
-    ) shouldBe "7.3.0 (Scala 3.x, 2.13), 7.2.0 (Scala 2.12), 7.1.0 (Scala 2.11)"
-  }
-
```

Comment:
To take advantage of this being a simple unit test, which can be run easily without external infrastructure like docker, I've moved it further down the file (along with a couple of other new unit tests) into `BadgesUnitTests` - this doesn't extend `ControllerBaseSuite` and so can be quickly run in IntelliJ, etc, without having to start docker.